### PR TITLE
Info API Wrapper Refactoring

### PR DESCRIPTION
**Pull request by Cyber-Freak999**
- Worked on error handling on each endpoint
- Refactored payload parameters

### CODE REVIEW

**Comment by uchibeke** [comment]
@thelimeskies could you also help review this?

@Cyber-Freak999 should this not be the same PR as the other python refactor?

**Review by thelimeskies — APPROVED** [review]

**Comment by Cyber-Freak999** [comment]
Didn't notice it during the last refactor

File: submissions/Chimoney-Python/chimoney/Info.py [hunk 1]
```diff
@@ -15,7 +15,7 @@ def airtime_countries(self) -> dict:
         """
         return self._handle_request("GET", "/v0.2/info/airtime-countries")
 
-    def assets(self, country_code: str = None) -> dict:
```
Comment (thesayfulla) [inline submissions/Chimoney-Python/chimoney/Info.py:18]: Hello, with `None` and `str` you can use `Optional` like this:
```py
x: Optional[str] = "something" if some_condition() else None
```

More about [here.](https://mypy.readthedocs.io/en/stable/cheat_sheet_py3.html#useful-built-in-types)

**Review by thesayfulla — CHANGES_REQUESTED** [review]

File: submissions/Chimoney-Python/chimoney/Info.py (hunk 1 above)
Comment (Cyber-Freak999) [reply submissions/Chimoney-Python/chimoney/Info.py:18]: I am not sure this is necessary in the mean time but I'll look into it
@thelimeskies what do you think?

**Review by uchibeke — APPROVED** [review]

**Pull request merged**